src/pages/PicsPage.js

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import Pics from '../components/Pics';
+import loading from '../assets/images/loading-progress.gif';
+
+const PicsPage = () => {
+  const [tempPictList, setTempPictList] = useState([]);
+  const { pics } = useSelector((state) => state.pics);
+  const picList = Object.keys(pics);
+
+  const handleClick = (event) => {
+    if (event.target.value !== 'both') {
+      const pepito = picList.filter((date) => (pics[date].media === event.target.value));
+      setTempPictList(pepito);
+    } else {
+      setTempPictList(picList);
+    }
+  };
+
+  useEffect(() => {
+    setTempPictList(picList);
+  }, [pics]);
+
+  return (
+
+    <div className="totallist">
+      <form action="/" className="form" id="form1">
+        <p>Choose media type: image, video or both (default)</p>
+        <select id="select" name="select" onChange={handleClick}>
+          <option value="both">Both</option>
+          <option value="image">Image</option>
+          <option value="video">Video</option>
+        </select>
```

Comment:
- [ ] Great job so far for adding options to select and display either an image, video, or both :+1: . Based on the requirements of this project kindly note that the filtering should be done on a search field. Imagine a search field where you can introduce the first letter of the title of an image/video and all images/videos that contain those letter(s) will be filtered at the top. Also, kindly note that you'll require a `Filter` stateless component to filter the data retrieved from the API.